botocore/docs/example.py

```diff
@@ -12,9 +12,12 @@
 # language governing permissions and limitations under the License.
 from botocore.docs.shape import ShapeDocumenter
 from botocore.docs.utils import py_default
+from botocore.model import get_streaming_body
 
 
 class BaseExampleDocumenter(ShapeDocumenter):
+    streaming_shape = None
```

Comment:
I would prefer this is not an attribute of the class. Much rather have it live in the constructor as an attribute of a ``ShapeDocumenter`` instance. If you are worried that ``streaming_shape ``is too specific, you can create a context attribute that is a dictionary that is used to store a bunch of this sort of metadata.